githooks/prepare-commit-msg

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/sh
+#
+# An example hook script to prepare the commit log message.
+# Called by "git commit" with the name of the file that has the
+# commit message, followed by the description of the commit
+# message's source.  The hook's purpose is to edit the commit
+# message file.  If the hook fails with a non-zero status,
+# the commit is aborted.
+#
+# To enable this hook, rename this file to "prepare-commit-msg".
+
```

Comment:
It's always better [to write safer shell scripts](https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/):
```suggestion
set -euxo pipefail
```